pypdf/_text_extraction/_layout_mode/_fixed_width_page.py

```diff
@@ -0,0 +1,398 @@
+"""Extract pdf text preserving the structural layout of the source PDF"""
```

Comment:
What would non-structural layout be?

Reply:
We'll need a philosopher to answer that one. ;)

Good point. Addressed.